### PR TITLE
python meterpreter chmod command

### PR DESCRIPTION
**Pull request by timwr**
See https://github.com/rapid7/metasploit-framework/pull/10575

### CODE REVIEW

File: python/meterpreter/ext_server_stdapi.py [hunk 1]
```diff
@@ -1326,6 +1328,13 @@ def stdapi_fs_file_copy(request, response):
     shutil.copyfile(unicode(oldname), unicode(newname))
     return ERROR_SUCCESS, response
 
+@register_function_if(sys.platform == 'darwin' or sys.platform.startswith('linux'))
```
Comment (phra) [inline python/meterpreter/ext_server_stdapi.py]: shouldn't `os.chmod` work also on windows machines?

Comment (timwr) [reply python/meterpreter/ext_server_stdapi.py]: I'm not sure of the behavior on windows: https://superuser.com/questions/106181/equivalent-of-chmod-to-change-file-permissions-in-windows

**Comment by wvu-r7** [comment]
I didn't see this PR and ended up writing my own, hah. I've just removed the Python commit.

File: python/meterpreter/ext_server_stdapi.py [hunk 2]
```diff
@@ -1326,6 +1328,13 @@ def stdapi_fs_file_copy(request, response):
...
+def stdapi_fs_chmod(request, response):
+    path = packet_get_tlv(request, TLV_TYPE_FILE_PATH)['value']
+    mode = packet_get_tlv(request, TLV_TYPE_FILE_MODE_T)['value']
+    os.chmod(path, mode)
```
Comment (wvu-r7) [inline python/meterpreter/ext_server_stdapi.py]: I had used `unicode(path)` here.

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-payloads/commit/2b26c69e20cc937b92c5ef041be264e06b787a38

**Comment by timwr** [comment]
Thanks @wvu!

**Pull request merged**